tools/ci/junit_upload.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# shellcheck source=/dev/null
+source /root/.bashrc
+# junit file name can differ in kitchen or macos context
+junit_files="junit-*.tar.gz"
+if [[ -n "$1" ]]; then
+    junit_files="$1"
```

Comment:
Does this only work if the first argument is in double quotes?

Reply:
I've tested locally and it seemed to be OK.
I'll create a fake commit to force generation of full tests (UT, E2E, etc) to double-check-validate in the CI as well, thanks